react/MinicartContext.tsx

```diff
@@ -62,10 +62,11 @@ const MinicartContextProvider: FC<Props> = ({
   children,
 }) => {
   const { isMobile } = useDevice()
+  const isPopup = variation === 'popup' || variation === 'popupWithLink'
```

Comment:
I suggest you create a predicate and reuse this popup validation. Something like that:

```tsx
function checkVariationIsPopup(variation) {
    return variation === 'popup' || variation === 'popupWithLink'
}
```

So this line would be 
```suggestion
const isPopup = checkVariationIsPopup(variation)
```

and at line 79 you can use 
```tsx
openBehavior:  checkVariationIsPopup(resolvedVariation)
```

Reply:
Hello Igor !

checkVariationIsPopup function created, could you validate, please?